docs/howto/system_testing.md

```diff
@@ -599,6 +599,80 @@ A sample workflow would look like:
 - Run `elastic-package stack up -d -v`
 - Navigate to the package folder in integrations and run `elastic-package test system -v`
 
+### Running system tests without cleanup (technical preview)
```

Comment:
Try to print also something about it being on technical preview on the command output when these flags are used.

Reply:
Added a message about the technical preview:

```
 $ elastic-package test system  --config-file data_stream/access/_dev/test/system/test-default-config.yml --setup
Run system tests for the package
Running tests per stages (technical preview)
2024/02/08 11:03:19  INFO Using service variant: ServiceVariant{Name: v1, Env: SERVICE_VERSION=1.19.5}
2024/02/08 11:03:21  INFO License text found in "/home/mariorodriguez/Coding/work/integrations/LICENSE.txt" will be included in package
```

Anything else to add to that message?

Reply:
Looks good.